docker-compose-build.yaml

```diff
@@ -3,22 +3,22 @@ services:
   batcher:
     build:
       context: .
-      dockerfile: disperser/batcher.Dockerfile
+      dockerfile: disperser/cmd/batcher/Dockerfile
     image: ghcr.io/layr-labs/eigenda/batcher:${BUILD_TAG:-latest}
   disperser:
     build:
       context: .
-      dockerfile: disperser/disperser.Dockerfile
+      dockerfile: disperser/cmd/apiserver/Dockerfile
     image: ghcr.io/layr-labs/eigenda/disperser:${BUILD_TAG:-latest}
```

Comment:
should we rename the image to apiserver?

Reply:
I think this would make sense.

Reply:
Will do this in a separate PR since it requires coordination with devops repo.